### PR TITLE
Fix address validation

### PR DESCRIPTION
**Pull request by erikh360**

### CODE REVIEW

File: uniqueids/tests.py [hunk 1]
```diff
@@ -760,7 +760,8 @@ def test_personnel_upload_good(self):
             {
                 'communicate_through': None,
                 'details': {
-                    'addresses': {'msisdn': {'+2347012312341': {"default": True}}},
+                    'addresses': {
```
Comment (OMosimege) [inline uniqueids/tests.py]: I'm assuming these were the missing environment variables that caused the previous fail?

**Review by OMosimege — APPROVED** [review]
Looks great Erik! 💯

**Review by OMosimege — APPROVED** [review]
everything looks great 💯

File: uniqueids/tests.py (hunk 1 above)
Comment (erikh360) [reply uniqueids/tests.py]: It was `flake8` complaining that the line was too long.

Comment (OMosimege) [reply uniqueids/tests.py]: okay great 👍

**Pull request merged**